File: 1436-destination-city/1436-destination-city.py
class Solution:
    def destCity(self, paths: List[List[str]]) -> str:
        s = set()
        for i in range(len(paths)):
            s.add(paths[i][0])
        for i in range(len(paths)):
            candi = paths[i][1]
            if candi not in s: return candi
        return ""

File: 1436-destination-city/1436-destination-city.cpp
class Solution {
public:
    string destCity(vector<vector<string>>& paths) {
        for(int i=0; i<paths.size(); i++){
            string candi = paths[i][1];
            bool good = true;
            for(int j=0; j<paths.size(); j++){
                if(paths[j][0] == candi){
                    good = false;
                    break;
                }
            }
            if(good) return candi;
        }
        return "";
    }
};